src/libs/ReportActionsUtils.js

```diff
@@ -170,6 +170,28 @@ function getMostRecentIOURequestActionID(reportActions) {
     return _.last(sortedReportActions).reportActionID;
 }
 
+/**
+ * Returns array of links inside given report
+ *
+ * @param {reportAction} reportAction
```

Comment:
```suggestion
 * @param {object} reportAction
```
there's no typedef for `reportAction` anywhere I don't think (well not yet, we would maybe have one with TS eventually)